dashboard/test/ui/features/code_tools/pythonlab/pythonlab_run_eyes.feature

```diff
@@ -0,0 +1,18 @@
+@no_mobile
+# Our minimum version of Safari does not support web workers
+@no_safari
+@eyes
```

Comment:
Out of curiosity, would an eyes test run on Safari anyway?

Reply:
Yeah I wasn't sure if this was needed...I copied from the ui tests, I can remove this

Reply:
I think I'll leave it, when you run locally it'll try to run against Safari so this makes it explicit.